SplitFiles/control.c

```diff
@@ -191,7 +188,7 @@ void execute_command(ControlBuffer *ctrlBuff, GameParameters* params) {
                 case 'B':
                 if (fetch_bread(e)) {
                     // brown takes longer
-                    new_slice->old_moisture = 96 + rand()%64;
+                    new_slice->moisture = 96 + rand()%64;
                     new_slice->thermalMass = 82;
```

Comment:
For smoother plotting these thermal mass numbers should probably be multiples of 16. Doesn't really matter much though.